apps/messages/src/webpack.config.ts

```diff
@@ -0,0 +1,41 @@
+import path from 'path';
+import { Configuration } from 'webpack';
+import TemplatePlugin from './template-plugin';
+
+const outputDir = path.resolve(
+  require.resolve('@asap-hub/messages/package.json'),
```

Comment:
`__dirname` maybe?

Reply:
Reasoning was that it's absolute within the package, not relative to where this file is if it is moved into a subdir of src or anything